### ISSUE TITLE
Download image failed NSURLErrorDomain Code=-999 

### ISSUE THREAD

**Issue by rango123456** [rs/SDWebImage#2325]
### New Issue Checklist

* [ ] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [ ] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios 
 Platform Version        | e.
 SDWebImage Version       |4.3.3 
 Integration Method     |cocoapods 
 Xcode Version            |Xcode 9
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | http://rongcloud-image.ronghub.com/image_jpeg__RC-2018-05-16_4327_1526451613667.jpg?e=2147483647&token=livk5rb3__JZjCtEiMxXpQ8QscLxbNLehwhHySnX:0boO7bphtDwa-pvvSOisUS8nQEI=

### Issue Description and Steps

his url can be accessed normally in the browser but cannot be displayed with SDWebImage

**Comment by rango123456** [comment]
this url can be accessed normally in the browser but cannot be displayed with SDWebImage
